### PR TITLE
Add workflow with cutadapt, STAR, and HT-seq

### PR DESCRIPTION
**Pull request by mzganec**

### CODE REVIEW

File: resolwe_bio/processes/workflows/cutadapt_star_htseq.yml [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+  input:
+    - name: reads
+      label: NGS reads
+      type: data:reads:fastq:single
+    - name: genome
+      label: Indexed reference genome
+      type: data:genomeindex:star
+      description: >
+        Genome index prepared by STAR aligner indexing tool
+      required: false
```
Comment (jkokosar) [inline resolwe_bio/processes/workflows/cutadapt_star_htseq.yml]: Genome (index) should be a required input.

File: resolwe_bio/processes/workflows/cutadapt_star_htseq.yml [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+  input:
+    - name: reads
+      label: NGS reads
+      type: data:reads:fastq:paired
+    - name: genome
+      label: Indexed reference genome
+      type: data:genomeindex:star
+      description: >
+        Genome index prepared by STAR aligner indexing tool
+      required: false
```
Comment (jkokosar) [inline resolwe_bio/processes/workflows/cutadapt_star_htseq.yml]: Genome (index) should be a required input.

**Comment by jkokosar** [comment]
Also, please rebase / fix issue with the changelog.

File: resolwe_bio/processes/expression/htseq_count.yml [hunk 3]
```diff
@@ -241,7 +241,7 @@
           samtools sort -n {{ alignments.bam.file }} "${NAME}_name_sorted"
           re-checkrc
           re-progress 0.5
-          htseq-count --format=bam --order=name --stranded={{ stranded }} --mode={{mode}} --type={{ feature_type }} --idattr={{ id_attribute }} ${NAME}_name_sorted.bam {{ gff.gtf.file }} > "HTseq_rc.txt"
+          htseq-count --format=bam --order=name --stranded={{ stranded }} --mode={{mode}} --type={{ feature_type }} --idattr={{ id_attribute }} "${NAME}_name_sorted.bam" {{ gff.gtf.file }} > "HTseq_rc.txt"
```
Comment (mstajdohar) [inline resolwe_bio/processes/expression/htseq_count.yml]: Split into multiple lines.

File: resolwe_bio/processes/expression/htseq_count.yml [hunk 4]
```diff
@@ -251,7 +251,7 @@
       re-progress 0.8
 
       head -n -5 HTseq_rc.txt > rc_wo_header.txt
-      expression_fpkm_tpm.R  {{ gff.gtf.file }} 'gtf' {{ feature_type }} {{ id_attribute }} 'rc_wo_header.txt'
+      expression_fpkm_tpm.R {{ gff.gtf.file }} 'gtf' {{ feature_type }} {{ id_attribute }} 'rc_wo_header.txt'
       re-checkrc
       re-progress 0.9
```
Comment (mstajdohar) [inline resolwe_bio/processes/expression/htseq_count.yml]: Not needed

**Comment by mstajdohar** [comment]
Use spaces in test file names.

File: resolwe_bio/processes/expression/htseq_count.yml [hunk 5]
```diff
@@ -104,7 +104,7 @@
           samtools sort -n {{ alignments.bam.file }} "${NAME}_name_sorted"
           re-checkrc
           re-progress 0.5
-          htseq-count --format=bam --order=name --stranded={{ stranded }} --mode={{mode}} --type={{ feature_type }} --idattr={{ id_attribute }} ${NAME}_name_sorted.bam {{ gff.gtf.file }} > "HTseq_rc.txt"
+          htseq-count --format=bam --order=name --stranded={{ stranded }} --mode={{mode}} --type={{ feature_type }} --idattr={{ id_attribute }} "${NAME}_name_sorted.bam" {{ gff.gtf.file }} > "HTseq_rc.txt"
         fi
       {% else %}
         htseq-count --format=bam --order=pos --stranded={{ stranded }} --mode={{mode}} --type={{ feature_type }} --idattr={{ id_attribute }} {{ alignments.bam.file }} {{ gff.gtf.file }} > "HTseq_rc.txt"
```
Comment (mstajdohar) [inline resolwe_bio/processes/expression/htseq_count.yml]: Split into multiple lines.

File: resolwe_bio/processes/expression/htseq_count.yml [hunk 6]
```diff
@@ -241,7 +241,7 @@
...
```
Comment (mstajdohar) [inline resolwe_bio/processes/expression/htseq_count.yml]: Split into multiple lines.

**Comment by jvrakor** [comment]
Split the PR into 3 separate commits

**Pull request merged**